src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Implementation/AppBase.cs

```
@@ -430,5 +466,121 @@ private List<string> GetOptionIdsFromFormLayout(string formLayout)
+                Type = "ConversationId",
+                ScopeInformation = new List<ScopeInformation>
+                    {
+                        new ScopeInformation
+                        {
+                            ExpectedResponseDateTime = completedTime.AddHours(2)
+                        }
+                    },
+            };
+
+            BusinessScope businessScope = new BusinessScope
+            {
+                Scope = new List<Scope> { scope }
+            };
+
+            DocumentIdentification documentIdentification = new DocumentIdentification
+            {
+                InstanceIdentifier = instanceGuid,
+                Standard = _appMetadata.EFormidling.Standard,
+                TypeVersion = _appMetadata.EFormidling.TypeVersion,
+                CreationDateAndTime = completedTime,
+                Type = _appMetadata.EFormidling.Type
+            };
+
+            StandardBusinessDocumentHeader sbdHeader = new StandardBusinessDocumentHeader
+            {
+                HeaderVersion = "1.0",
+                BusinessScope = businessScope,
+                DocumentIdentification = documentIdentification,
+                Receiver = receivers,
+                Sender = new List<Sender> { digdirSender }
+            };
+
+            // Skal sikkerhetsnivå kunne konfigureres i appen? 
+            StandardBusinessDocument sbd = new StandardBusinessDocument
+            {
+                StandardBusinessDocumentHeader = sbdHeader,
+                Arkivmelding = new Arkivmelding { Sikkerhetsnivaa = 3 },
+            };
+
+            return sbd;
+        }
+
+        private async void SendEFormidlingShipment(Instance instance, string taskId)
+        {
+            string instanceGuid = instance.Id.Split("/")[1];
+
+            StandardBusinessDocument sbd = await ConstructStandardBusinessDocument(instanceGuid);
+            StandardBusinessDocument sbdVerified = await _eFormidlingClient.CreateMessage(sbd);
```

What do we think of error handling here? 
Should we let the app crash or log error and let it continue silently? I guess we have many other options as well.

---

The error is swallowed by ProcessController and the result frontend is "Ukjent feil"